### PR TITLE
[Improvement] Refactor frontend of job query page

### PR DESCRIPTION
**Pull request by s7monk**
close: https://github.com/apache/paimon-webui/issues/441

### Purpose

Refactor the job query page frontend to support submitting multiple jobs at the same time.

image

image

### CODE REVIEW

File: paimon-web-ui/src/store/job/index.ts [hunk 1]
```diff
@@ -15,89 +15,139 @@ KIND, either express or implied.  See the License for the
...
-      if (this.currentJob && this.currentJob.resultData && this.currentJob.resultData.length > 0)
-        return Object.keys(this.currentJob.resultData[0]).length
-      else if (this.jobResultData && this.jobResultData.resultData && this.jobResultData.resultData.length > 0)
-        return Object.keys(this.jobResultData.resultData[0]).length
-      else
-        return 0
+    getJobStatus(state): (key: string) => string {
+      return (key) => {
+        if (!state.jobs[key]) {
+          return ''
```
Comment (zhangmo8) [inline paimon-web-ui/src/store/job/index.ts:46]: Set a valid default value

File: paimon-web-ui/src/store/job/index.ts [hunk 2]
```diff
@@ -15,89 +15,139 @@ KIND, either express or implied.  See the License for the
...
+      return (key) => {
+        const jobDetails = state.jobs[key]
+        if (jobDetails) {
+          if (jobDetails.job && jobDetails.job.resultData && jobDetails.job.resultData.length > 0) {
+            return Object.keys(jobDetails.job.resultData[0]).length
+          }
+          if (jobDetails.jobResultData && jobDetails.jobResultData.resultData && jobDetails.jobResultData.resultData.length > 0) {
+            return Object.keys(jobDetails.jobResultData.resultData[0]).length
+          }
+        }
```
Comment (zhangmo8) [inline paimon-web-ui/src/store/job/index.ts:72]: You can use the optional chain to reduce the length of judgment.

File: paimon-web-ui/src/store/job/index.ts [hunk 3]
```diff
@@ -15,89 +15,139 @@ KIND, either express or implied.  See the License for the
...
+      return (key) => {
+        const jobDetails = state.jobs[key]
+        if (jobDetails) {
+          if (jobDetails.job && jobDetails.job.resultData && jobDetails.job.resultData.length > 0) {
+            return jobDetails.job.resultData.length
+          }
+          if (jobDetails.jobResultData && jobDetails.jobResultData.resultData && jobDetails.jobResultData.resultData.length > 0) {
+            return jobDetails.jobResultData.resultData.length
+          }
+        }
```
Comment (zhangmo8) [inline paimon-web-ui/src/store/job/index.ts:86]: You can use the optional chain to reduce the length of judgment.

**Review by zhangmo8 — COMMENTED** [review]
Licenses that need to be supplemented with new dependencies

**Comment by s7monk** [comment]
> Licenses that need to be supplemented with new dependencies

done.

**Review by zhangmo8 — APPROVED** [review]
LGTM

**Review by SteNicholas — APPROVED** [review]

**Pull request merged**